src/TwigFractal.php

```diff
@@ -1,20 +1,48 @@
 <?php
+/**
+ * @file
+ * Contains Drupal\twig_fractal\TwigFractal.
+ */
 
 namespace Drupal\twig_fractal;
 
 use Drupal\twig_fractal\TokenParser\Render;
 use Twig_Extension;
 
+/**
+ * Registers a 'render' tag to make Fractal components reusable in Drupal 8.
+ *
+ * Example usage:
+ * @code
+ * {% render '@foo/baz.twig' %}
+ * {% render '@foo/baz.twig' with { qux: 'foobar' } %}
+ * {% render '@foo/baz--bar.twig' with { qux: 'foobar' } %}
+ * @endcode
+ *
+ * @implements Twig_ExtensionInterface
+ */
```

Comment:
I mean that this class is basically a wrapper to introduce the functionality of the `Drupal\twig_fractal\TokenParser\Render` so maybe in the class level description makes sense to some reference to it. Anyway I'm not sure if this make sense, so just sharing my thinks as a 'developer how just read this file'